x/ibc/light-clients/07-tendermint/types/upgrade_test.go

```diff
@@ -478,6 +430,16 @@ func (suite *TendermintTestSuite) TestVerifyUpgrade() {
 		suite.SetupTest()
 
 		clientA, _ = suite.coordinator.SetupClients(suite.chainA, suite.chainB, exported.Tendermint)
+		upgradedClient = types.NewClientState("newChainId", types.DefaultTrustLevel, trustingPeriod, ubdPeriod+trustingPeriod, maxClockDrift, newClientHeight, commitmenttypes.GetSDKSpecs(), upgradePath, false, false)
+		upgradedClient = upgradedClient.ZeroCustomFields()
```

Comment:
Why don't you just zero the custom fields to begin with?

Reply:
ah yea I can do this